x/dsmr/p2p.go

```diff
@@ -127,10 +128,18 @@ func (c ChunkSignatureRequestVerifier[T]) Verify(
 		}
 	}
 
+	// verify that the signature is a valid signature over the unsigned chunk.
```

Comment:
Could we remove obvious comments that match the function name ie. parse and verify unless the comment provides additional information?

Reply:
sure.